src/pybind/mgr/orchestrator.py

```diff
@@ -391,6 +391,10 @@ def remove_stateful_service_rule(self, service_type, id_):
         """
         raise NotImplementedError()
 
+    def host_ls(self):
```

Comment:
have a look at `def get_hosts(self)` instead.

Reply:
Done.

Reply:
33e6f39b2f7d7a7b40ace7e3ca8f0a55dabcb865 is already merged into mater. You can safely remove this now

Reply:
mmm, this is rebased and "no-conflicts" present... probably this will not not affect/modify the code present in master branch at all.

Reply:
Ok... i see ... now host_ls is get_hosts